addons/mail/static/src/core/web/mail_core_web_service.js

```diff
@@ -6,13 +6,14 @@ import { registry } from "@web/core/registry";
 
 export class MailCoreWeb {
     constructor(env, services) {
-        Object.assign(this, {
-            rpc: services.rpc,
-        });
+        /** @type {ReturnType<typeof import("@bus/services/bus_service").busService.start>} */
+        this.busService = services["bus_service"];
         /** @type {import("@mail/core/common/message_service").MessageService} */
         this.messageService = services["mail.message"];
         /** @type {import("@mail/core/common/messaging_service").Messaging} */
         this.messagingService = services["mail.messaging"];
+        /** @type {ReturnType<typeof import("@web/core/network/rpc_service").rpcService.start} */
```

Comment:
Missing `>` in-between `start` and `}` at the end, although VSCode seems to resolve correctly even with the typo